backend/api/python_http_client_template/partial_header.mustache

```diff
@@ -0,0 +1,14 @@
+"""
+{{#appName}}
+    {{{appName}}}
+{{/appName}}
+
+{{#appDescription}}
+    {{{appDescription}}}
+{{/appDescription}}
+
+    {{#infoEmail}}
+    Contact: {{{infoEmail}}}
+    {{/infoEmail}}
+    Generated by: https://openapi-generator.tech
+"""
```

Comment:
This is the template override. Original template at https://github.com/OpenAPITools/openapi-generator/blob/v4.3.1/modules/openapi-generator/src/main/resources/python/partial_header.mustache